cmd/containerd-shim-runhcs-v1/service_internal.go

```diff
@@ -104,6 +104,18 @@ func (s *service) createInternal(ctx context.Context, req *task.CreateTaskReques
 		return nil, errors.Wrap(err, "unable to process OCI Spec annotations")
 	}
 
+	// If sandbox isolation is set to hypervisor, make sure the HyperV option
+	// is filled in. This lessens the burden on Containerd to parse our shims
+	// options if we can set this ourselves.
+	if shimOpts.SandboxIsolation == runhcsopts.Options_HYPERVISOR {
+		if spec.Windows == nil {
+			spec.Windows = &specs.Windows{}
```

Comment:
Even for LCOW?

Reply:
Looks like old code did do that lol. i dont remember why

Reply:
We identify LCOW by the Linux section being filled in as well as hyperv. Which kinda makes sense in my head hahah

Reply:
Oh, I am dumb. yes I know why this works lol. I for some reason thought that `spec.Windows.HyperV` was outside of `spec.Windows` and you didnt need the `Windows` part to add the `HyperV` part. Just ignore me.